open-media-match/src/OpenMediaMatch/app.py

```diff
@@ -66,6 +66,9 @@ def status():
     # Linters complain about imports off the top level, but this is needed
     # to prevent circular imports
 
+    if not os.environ.get("PRODUCTION", False) and app.config.get("ROLE_HASHER", False) and app.config.get("ROLE_MATCHER", False):
```

Comment:
We might relax these requirements over time, since we might put non hash/match helpers in development.